### ISSUE TITLE
Resize windows

### ISSUE THREAD

**Issue by DanielAndreasen** [qtile/qtile#4367]
### The issue:

This is probably just me being a bit confused. I change between two layouts, MonadTall and Bsp. When I have multiple windows open and want to resize the current window, I would like to do something like: `mod+ctrl+l` to expand towards left, and similar for hjk (vim-keybindings). But how do I configure that?
I already have something that almost works, but only for one type of layout, e.g. for MonadTall:
```python
Key([mod, "control"], "h", lazy.layout.grow(), desc="Grow window to the left"),
Key([mod, "control"], "l", lazy.layout.shrink(), desc="Grow window to the right"),
```

### Required:

- [X] I have searched past issues to see if this bug has already been reported.

**Comment by elParaguayo** [comment]
You can have multiple commands per key. You can also limit when a command is run e.g.
```python
Key(
    [mod, "control"],
    "h",
    lazy.layout.grow().when(layout="monadtall"),
    lazy.layout.grow_left().when(layout="bsp"),
    desc="Grow window to the left"
),
```

**Comment by elParaguayo** [comment]
You don't need to add then `.when(...)` part but, if you don't, you'll get some error messages in your log file if the layout doesn't have the relevant command name.

Ideally, we'd have consistent names across layouts but I'm not sure how feasible that is.

**Comment by DanielAndreasen** [comment]
Yes, I can see the challenge in that. I think this is a decent solution given the circumstances.
